tests/unittests/Foundation/NSRecursiveLockTests.m

```diff
@@ -20,143 +20,6 @@
 #import <thread>
 #import <mutex>
 #import <condition_variable>
-#import <chrono>
-
-#import "NSRecursiveLock+Internal.h"
```

Comment:
_lockCount is our private function, so unfortunately not :(

---
In reply to: [72315210](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72315210) [](ancestors = 72315210)